cmsis/TARGET_CORTEX_A/cmsis_compiler.h

```diff
@@ -0,0 +1,211 @@
+/**************************************************************************//**
```

Comment:
We carry both, that's why I think it's better to have RTX<num> but for CMSIS it doesn't really matter and adding 5 may be confusing.